docs/development/config-files.md

```diff
@@ -399,3 +399,49 @@ Properties listed before the `$import` statement may get overwritten by the impo
 	"template": true
 }
 ```
+
+### Conditional configuration settings
+
+For devices that only slighly change between firmware versions or hardware revisions, you can enable some settings conditionally with the `"$if"` property. It accepts a string with logic inside, e.g.
+
+```json
```

Comment:
I suggest use to use js instead of json so the syntax highlight doesn't highlight comments

Reply:
That's just here in the diff. Github knows the type `jsonc` where it does not complain, but Docsify doesn't. However it renders json with comments just fine:

![grafik](https://user-images.githubusercontent.com/17641229/108388360-d85cc880-720e-11eb-84a3-8f0a7e232ecf.png)